packages/server-wallet/src/db/migrations/20201201095501_ledger_proposals.ts

```diff
@@ -0,0 +1,27 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<any> {
+  await knex.schema.table('channels', function(table) {
```

Comment:
request: instead of placing two-party-oriented data on `channels`, there ought to be a new `ledger_proposals` table with this rough schema:
```
ledger_id: foreign key to channels
proposal: jsonb
nonce: integer
signing_address: text
```

this allows the DB to store data for `n`-party ledger channels (such as the joint channel in a virtual channel construction!)